File: 1528-kids-with-the-greatest-number-of-candies/solution.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        ArrayList<Boolean> extra=new ArrayList<>();
        int max=0;
        for(int i=0;i<candies.length;i++)
        {
            if(candies[i]>max)
            max=candies[i];
        }
        for(int i=0;i<candies.length;i++)
        {
            if(candies[i]+extraCandies>=max)
            extra.add(true);
            else
            extra.add(false);
        }
        LinkedList<Boolean> list=new LinkedList<>(extra);
        return list;
    }
}

File: 1528-kids-with-the-greatest-number-of-candies/solution.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        result=[]
        x=max(candies)
        for i in candies:
            if i+extraCandies>=x:
                result.append(True)
            else:
                result.append(False)
        return result
